src/3d/castlecameras.pas

```diff
@@ -1378,6 +1383,30 @@       TExamineVectors = record
       read FRotationAccelerate write SetRotationAccelerate default true;
   end;
 
+  TCastle2DNavigation = class (TCastleExamineNavigation)
+  public
+    constructor Create(AOwner: TComponent); override;
+
+    { Drag with this mouse button to move the model. }
+    property MouseButtonMove: TCastleMouseButton
+      read FMouseButtonMove write FMouseButtonMove default buttonRight;
+    { Drag with this mouse button to zoom the model (look closer / further). }
+    property MouseButtonZoom: TCastleMouseButton
+      read FMouseButtonZoom write FMouseButtonZoom default buttonMiddle;
+
+    { Optional movement algorithm, moves the camera exactly as many units as
+      the mouse position has changed especially useful in 2D in the editor. }
+    property ExactMovement: Boolean read FExactMovement write FExactMovement default true;
+  published
+    { Enable rotating the camera around the model by user input.
+      When @false, no keys / mouse dragging / 3D mouse etc. can cause a rotation.
+
+      Note that this doesn't prevent from rotating by code, e.g. by setting
+      @link(Rotations) property or calling @link(SetView). }
+    property RotationEnabled: Boolean read FRotationEnabled write FRotationEnabled default false;
```

Comment:
This declares a new property `RotationEnabled` that "obscures" the property `RotationEnabled` at ancestor.

To only change the default value of ancestor property, use

```
property RotationEnabled default false;
```

(without type specified).

Same goes for `MouseButtonMove`, `MouseButtonZoom`.

Reply:
Same for `ExactMovement`. My initial comments around it were invalid. TCastleExamineNavigation implements the `ExactMovement`, and TCastle2DNavigation just makes it by default true in your PR, this it will have 100% backward compat.

Reply:
Fixed, also removed comments as you suggested.